mantidproject/mantid
# Indirect Inelastic Fixed Window Scan

**Description of work.**
Fixes the peak integration in `IndirectILLReductionFWS`.
It is now taking the peak positions from the monitor as opposed to before, when it was erroneously taking them from the first detector.

**To test:**
Download this file, rename to `253023.nxs`, supply to the algorithm below:
[253023.nxs.zip](https://github.com/mantidproject/mantid/files/3669430/253023.nxs.zip)

```
ifws = IndirectILLReductionFWS(Run='253023.nxs')
```

Check the sample logs of the output.
There should be entries for `MonitorLeftPeak` and `MonitorRightPeak` with values 2 and 252 respectively.



Fixes #26901 . 



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
